wuvt/templates/admin/article_edit.html

```diff
@@ -94,6 +94,20 @@ <h1>Edit Article</h1>
 {% endfor %}
         </div>
     </div>
+<div class="form-group{% if form.pinned_article.errors|length > 0 %} has-error{% endif %}">
+        <div class="col-sm-3">
+        </div>
+        <div class="col-sm-9 checkbox">
+            <label>
+                <input type="checkbox" name="pinned_article" id="id_pinned_article" value="{{ article.pinned_article }}" {% if article.pinned_article %}checked{% endif %}" />
```

Comment:
There's a typo here. This generates HTML that looks like this:
```<input type="checkbox" name="pinned_article" id="id_pinned_article" value="True" checked"```
Note the trailing quote; this should be removed.

This is nitpicky, but if the article is not pinned, this adds an extra space. Maybe it would be better to move the conditional before the space and only add the space if the checkbox is checked, e.g. ```%} checked```